imports/plugins/core/orders/client/components/orderBulkActionsBar.js

```diff
@@ -0,0 +1,179 @@
+              value="picked"
+              onClick={this.handleListItemClick}
+              listItemClassName={this.state.shipping.picked ? "selected" : ""}
+            >
+              {this.state.shipping.picked ?
+                <div>
+                  <Icon className="bulk-actions-icons" icon="fa fa-print"/>
+                  <Icon className="bulk-actions-icons-select" icon="fa fa-check"/>
+                </div>
+                :
+                this.renderLoadingSpinner("picked")
+              }
+            </ListItem>
+            <ListItem
+              label="Packed"
+              i18nKeyLabel="order.packed"
+              value="packed"
+              onClick={this.handleListItemClick}
+              listItemClassName={this.state.shipping.packed ? "selected" : ""}
+            >
+              {this.state.shipping.packed ?
+                <Icon className="bulk-actions-icons-select" icon="fa fa-check"/>
+                :
+                this.renderLoadingSpinner("packed")
+              }
+            </ListItem>
+            <ListItem
+              label={this.state.shipping.labeled ? "Generate Label" : "Labeled"}
+              i18nKeyLabel={this.state.shipping.labeled ? "order.generateLabel" : "order.labeled"}
+              value="labeled"
+              onClick={this.handleListItemClick}
+              listItemClassName={this.state.shipping.labeled ? "selected" : ""}
+            >
+              {this.state.shipping.labeled ?
+                <div>
+                  <Icon className="bulk-actions-icons" icon="fa fa-print"/>
+                  <Icon className="bulk-actions-icons-select" icon="fa fa-check"/>
+                </div>
+                :
+                this.renderLoadingSpinner("labeled")
+              }
+            </ListItem>
+            <ListItem
+              label="Shipped"
+              i18nKeyLabel="order.shipped"
+              value="shipped"
+              onClick={this.handleListItemClick}
+              listItemClassName={this.state.shipping.shipped ? "selected" : ""}
+            >
+              {this.state.shipping.shipped ?
+                <div>
+                  <Icon className="bulk-actions-icons" icon="fa fa-paper-plane-o" />
+                  <Icon className="bulk-actions-icons-select" icon="fa fa-check"/>
+                </div>
+                :
+                this.renderLoadingSpinner("shipped")
+              }
+            </ListItem>
+          </List>
+        );
+      }
+    }
+
+    handleListItemClick = (event, value) => {
+      if (this.props.setShippingStatus) {
+        this.props.setShippingStatus(value, this.props.selectedItems);
+      }
+    }
+
+    setListItemsToDefault() {
+      if (this.state.renderFlowList === false) {
+        this.setState({
+          shipping: {
+            picked: false,
+            packed: false,
+            labeled: false,
+            shipped: false
```

Comment:
How does this work if there is a shipping workflow that has additional or fewer steps?